src/System.Runtime.Serialization.Xml/tests/XmlDictionaryWriterTest.cs

```diff
@@ -107,7 +107,8 @@ public static void XmlBaseWriter_CheckAsync_ThrowInvalidOperationException()
             var t2 = Assert.ThrowsAsync<InvalidOperationException>(() => writer.WriteBase64Async(bytes, 0, byteSize));
 
             InvalidOperationException e = t2.Result;
-            Assert.StrictEqual(e.Message, "An asynchronous operation is already in progress.");
+            bool isAsyncIsRunningException = e.Message.Contains("XmlAsyncIsRunningException") || e.Message.Contains("in progress");
```

Comment:
Thanks for the suggestion, @davidsh . 

I checked System.Net.WebSockets.Client's tests. That test project includes the actual .resx file used by the library. For serialization, this seems meaning we would also need to change the ToF wrapper project to also include the .resx file. Do we want to make this change? Or should we just skip this message check? @SGuyGe @khdang